client/src/components/Button.tsx

```diff
@@ -2,8 +2,13 @@ import './Button.css';
 
 type Prop = {
   text: string;
+  onClick?: () => void;
```

Comment:
Interesting that this is optional. I've yet to see a button that wasn't intended to be clicked. :-)